src/priiloader/hacks_hash.ini

```diff
@@ -249,6 +249,45 @@ offset=0x8137ad0c
 patch=0x4bf840f4
 offset=0x812fef00
 patch=0x815488f8,0x8137ad10,0x00000000,0x00000000
+[Wiimmfi Patch v4 for vWii 5.2.0J]
```

Comment:
please change the title to 'Wiimmfi patch v4' and add a comment about the vwii version. that extra info is useless for users. this applies to all hacks

Reply:
Will do that.